lib/datadog/profiling/crashtracker.rb

```diff
@@ -13,6 +13,14 @@ module Profiling
     #
     # Methods prefixed with _native_ are implemented in `crashtracker.c`
     class Crashtracker
+      LIBDATADOG_API_FAILURE =
+        begin
+          require "libdatadog_api.#{RUBY_VERSION[/\d+.\d+/]}_#{RUBY_PLATFORM}"
+          nil
+        rescue LoadError => e
+          e.message
+        end
```

Comment:
This is needed for gracefully handling the extension being missing. Before the extraction, we were relying on the profiler loading code taking care of the graceful loading (which it does)